languages/golang/golang.go

```diff
@@ -21,26 +33,22 @@ func (l language) GetLanguage() *sitter.Language {
 }
 
 func (l language) Imports(content []byte, n *sitter.Node) ([]string, error) {
+	once.Do(func() { q, qerr = sitter.NewQuery([]byte(query), l.GetLanguage()) })
```

Comment:
I've just move this compilation to init (to get rid of once), but I still wonder what's better - wdyt?
pros/cons

Reply:
Let's go with `init` + `panic`, as discussed in Slack.